docs/topics/auth/customizing.txt

```diff
@@ -296,6 +296,24 @@ checks if a user may close tasks::
 
     user.has_perm('app.close_task')
 
+Custom permissions may contain the ``%(model_name)s`` and ``%(verbose_name)s``
+placeholders. Child models will then inherit the specific permission::
```

Comment:
We should make clear that `model_name` is supported in `codename` and `verbose_name` in `name`.